activesupport/lib/active_support/core_ext/date_time/calculations.rb

```diff
@@ -139,14 +146,32 @@ def end_of_minute
   end
   alias :at_end_of_minute :end_of_minute
 
-  # Adjusts DateTime to UTC by adding its offset value; offset is set to 0.
+  # Returns a <tt>Time</tt> instance of the simultaneous time in the system timezone.
```

Comment:
Thanks for the quick reply! The additional context helps and I appreciate that you plan to look at it as part of your other work.

In my particular case it happens to be some test-only code that does the `.utc.rfc3339` call so I can easily adjust things on my side. Primarily raised it in case it's a possible issue for others.